ckanext/canada/i18n/en/LC_MESSAGES/ckanext-canada.po

```diff
@@ -6133,3 +6133,5 @@ msgstr "Keep me logged in"
 #~ msgid "time_descend"
 #~ msgstr "Newest first"
 
+msgid "Date format incorrect. Expecting YYYY-MM-DD"
+msgstr "Date format incorrect. Expecting YYYY-MM-DD"
```

Comment:
These changes won't do anything. `msgstr ""` or a missing message means "leave it as it is"